backend/src/main/kotlin/com/group7/artshare/DTO/DiscussionPostDTO.kt

```diff
@@ -11,7 +11,7 @@ class DiscussionPostDTO{
     var posterId : Long? = null
```

Comment:
this posterId field was removed in previous PR. Here it is forgotten to delete I guess, it would be nice if you remove it from DTO.

Reply:
Done